src/include/daos_pipe.h

```diff
@@ -0,0 +1,257 @@
+/**
+ * (C) Copyright 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+#ifndef __DAOS_PIPE_H__
+#define __DAOS_PIPE_H__
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+
+/** logic functions */
+#define DAOS_FILTER_FUNC_EQ		0
+#define DAOS_FILTER_FUNC_NE		1
+#define DAOS_FILTER_FUNC_NE		2
+#define DAOS_FILTER_FUNC_LT		3
+#define DAOS_FILTER_FUNC_LE		4
+#define DAOS_FILTER_FUNC_GE		5
+#define DAOS_FILTER_FUNC_GT		6
+#define DAOS_FILTER_FUNC_LIKE		7
+#define DAOS_FILTER_FUNC_ISNULL		8
```

Comment:
extra tab?

Reply:
That is weird. If you check the file in the web viewer/editor, the tabulation is correct. VIM also shows correct tabulation. For some reason, it shows an extra tab here.

Reply:
Just look here: https://github.com/daos-stack/daos/blob/eberroca_newAPI_pipelines/src/include/daos_pipe.h

Reply:
yea OK, ive seen github render things differently in PRs, so just making sure